skt/reporter.py

```diff
@@ -415,27 +426,35 @@ def getreport(self):
             msg += self.gettested()
             msg += self.getjobresults()
 
+        msg += ['\nPlease review the above outcome and reply to this email in'
+                ' case',
+                'it\'s deemed incorrect.',
+                '\nSincerely,',
+                '  Kernel CI Team']
```

Comment:
In general, we can't really tell people what to do, so better just be helpful. Also, an outcome cannot really be incorrect, it's just a fact, and we're not passing any judgment here to object to. We can also put a "thank you" here.

Perhaps like this:

    Please reply to this message, if you wish to report an issue with our build
    setup or our tests, suggest an improvement, or exclude yourself from
    receiving these messages.

    Thank you for your contribution to the Linux kernel!

    Sincerely,
    Kernel CI Team,
    Red Hat

And, yeah, we'll need to provide a way for people to stop receiving the messages.

Reply:
I wanted to put the positive thing (thanking) at the beginning of the message so it doesn't sound too sarcastic in case everything failed.

Definitely agree with adding a way to exclude people but since we don't have it implemented yet, we shouldn't be advertising it in the message. We can add it as it's implemented.

Reply:
I'm unsure about the thanking in general. I'm not sure if we're in the position to be thanking for the contribution to the Linux kernel directly, as we (Red Hat) don't own it. I.e. the sender of the patch is not sending it to us, therefore we can't really thank them. Let me think about that a little.

Regarding excluding, I think we *need* to put it in, especially at the start, when our reports and tests will be the worst. People need to know they're free to not receive those and we are not forcing them. It will help them see the reports better and accept them easier.

For now we can just respond to those e-mails ourselves, and quickly add an address filter to sktm setup when/if somebody asks that. It will also be acceptable to ask people to wait a little, while we implement that, as long as we promise and actually do that.

Reply:
I still think we should not ask people to read the report, as it would imply that we require them to heed it. Instead we should make it clear what it contains from the start, make it easy to read and understand, and give people the opportunity to use it, as a help. But we should in no way insist or even hint that they should.

How about this:

    Please reply to this message if you find an issue with our testing
    process, or wish to stop receiving these reports.

    Sincerely,
    Kernel CI Team,
    Red Hat

We can't really write "reply to these message if you want to start receiving
them again" as they shouldn't be getting them then. Instead I think it's OK to
let people know how to re-subscribe in response to their unsubscribe request.
Something like this:

    We're sorry to know you didn't find our test reports useful. We're working
    hard to improve them, and would be glad to hear what you found unsuitable
    and receive any suggestions you might have.

    If you wish to start receiving these reports again, simply send us a
    message at kernel-ci-list@redhat.com.